core/src/actors/blocks_manager/messages.rs

```diff
@@ -9,3 +13,13 @@ pub struct GetHighestBlockCheckpoint;
 impl Message for GetHighestBlockCheckpoint {
     type Result = ChainInfoResult<Epoch>;
 }
+
+/// Add a new block
+pub struct AddNewBlock {
```

Comment:
Just for the record: at this point we don't know whether the block will be added to the chain at all—we may change this at some point to `ProcessBlockCandidate`.